### ISSUE TITLE
Quiet Broadcast().

### ISSUE THREAD

**Issue by FullBleed** [RPTools/maptool#2644]
_**Is your feature request related to a problem? Please describe.**_

The default chat update sound ("BEEP!") that a broadcast() triggers steps on any sound effect that you may want to accompany the broadcast.  I keep running into this issue and I'm having to remove sounds or avoid adding cool sound effects to my game because I know the broadcast is going to ruin it.

Keep in mind that you don't realize the extent of the problem until you've got clients connected and the chat update BEEP is going off.     

_**Describe the solution you'd like**_

I suspect that the simplest path would be to add a new (0,1)  parameter for quiet output.  Defaulting to current behaviour.

_**Describe alternatives you've considered**_

Alternatives:
A) Have a new function like quietBroadcast() that will broadcast quietly.
B) Have a new audio parameter added to the existing broadcast to playClip()/playStream() that will accompany the broadcast and replace the chat update beep.  This might be convenient, but may not be the best option for people who have custom sound functions that they'd prefer to use.

**Comment by FullBleed** [comment]
> There is a Preference setting to turn off the chat notification sound.
> 
> I've always kept that turned off because it is really annoying.

Yes, I know about that setting.  I knew I should have mentioned it in the OP,  ;)

Turning system sounds off is a baby-out-with-the-bathwater solution.

While I wouldn't mind being to able to set different sounds for different chat output, losing that default sound for many things would be detrimental to my games.  Since I play with cameras, I can see how players learn to have a pavlovian response to that sound and bring their attention to what's going on in the chat window.  Without it, the less attentive will have to be prompted, and then we'll all have to sit while they "catch-up", etc.  Init sound updates, for example, are important... so, in general, things I broadcast I want some kind of audio notice for so people see it (or I wouldn't have the framework broadcasting it!)

In short--If I/they toggle off system sounds (I assume it's per client) I'd have to create an entire audio notification system for my framework the hard way by going in and adding to all my broadcasts.  I don't have a problem with MT's default sounds (in general.)  And I didn't have a problem with Broadcast forcing one on me before I could add my own (it's feature, not a bug!)  But now that someone can do something cool and interactive on screen that may produce chat output *and* sound, it's a problem.

_As an aside, I would be interested in how many people turn off system sounds in-game and use other techniques to command player attention during a game.  Can we do polls in discord?_

**Comment by Phergus** [comment]
AFAIK, Discord doesn't have native polling yet.  Most folks fake it by Posting a question and then immediately adding emojis under it for the responses:
![image](https://user-images.githubusercontent.com/34379254/117893230-4990a280-b277-11eb-89fe-cd30f3577ad2.png)

**Comment by bubblobill** [comment]
But now we do have surveys. You can ask the world.
